ts-obs/cmd/chunkIntervalReset.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+    "context"
+    "errors"
+	"fmt"
+    "os"
+
+	"github.com/spf13/cobra"
+    "github.com/jackc/pgx/v4/pgxpool"
+)
+
+// chunkIntervalResetCmd represents the chunk-interval reset command
+var chunkIntervalResetCmd = &cobra.Command{
+	Use:   "reset <metric>",
+	Short: "Resets the chunk interval for a specific metric back to the default",
+	RunE:  chunkIntervalReset,
+}
+
+func init() {
+	chunkIntervalCmd.AddCommand(chunkIntervalResetCmd)
+}
+
+func chunkIntervalReset(cmd *cobra.Command, args []string) error {
+    var err error
+
+    if len(args) != 1 {
+        return errors.New("\"ts-obs metrics reset-metric-chunk-interval\" requires 1 arguments")
+    }
```

Comment:
Cobra has a way of checking arguments for you, check out (https://github.com/spf13/cobra#positional-and-custom-arguments, and the example bellow it with `cobra.MinimumNArgs(1)`